src/v/utils/base64.h

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2020 Vectorized, Inc.
+ *
+ * Use of this software is governed by the Business Source License
+ * included in the file licenses/BSL.md
+ *
+ * As of the Change Date specified in that file, in accordance with
+ * the Business Source License, use of this software will be governed
+ * by the Apache License, Version 2.0
+ */
+#pragma once
+#include "bytes/bytes.h"
+#include "bytes/iobuf.h"
+#include "seastarx.h"
+
+#include <seastar/core/sstring.hh>
+
+class base64_decoder_exception : public std::exception {
```

Comment:
final? - same for all the exceptions so the linker eliminates the parent class.

Reply:
yeh good call